great_expectations/data_context/types/base.py

```diff
@@ -1206,9 +1206,6 @@ def __init__(
             commented_map = CommentedMap()
         self._commented_map = commented_map
         self._config_version = config_version
-        if datasources is None:
-            datasources = {}
-        self.datasources = datasources
```

Comment:
`
if datasources is None:	
            datasources = {}	
        self.datasources = datasources
`